zeppelin-zengine/pom.xml

```diff
@@ -42,6 +42,18 @@
     </dependency>
 
     <dependency>
+      <groupId>${project.groupId}</groupId>
```

Comment:
I don't think it is a good idea to make zengine to depend on one specific interpreter.